roles/sap_netweaver_preconfigure/README.md

```diff
@@ -1,21 +1,24 @@
 # sap_netweaver_preconfigure Ansible Role
 
 This role installs additional required packages and performs additional configuration steps for installing and running SAP NetWeaver.
-If you want to configure a RHEL system for the installation and later usage of SAP NetWeaver, you have to first run role sap_general_preconfigure
-and then role sap_netweaver_preconfigure.
+If you want to configure a RHEL system for the installation and later usage of SAP NetWeaver, you have to first run role `sap_general_preconfigure` and then role sap_netweaver_preconfigure.  
+For SLES, running the `sap_general_preconfigure` role is not necessary.
 
 ## Requirements
 
 To use this role, your system needs to be configured with the basic requirements for SAP NetWeaver or SAP HANA. This is typically done by
-running role sap_general_preconfigure (for RHEL managed nodes before RHEL 7.6, community maintained role sap-base-settings can be used).
+running role `sap_general_preconfigure` (for RHEL managed nodes before RHEL 7.6, community maintained role sap-base-settings can be used).
 It is also strongly recommended to run role linux-system-roles.timesync for all systems running SAP NetWeaver, to maintain an identical system time,
 before or after running role sap_netweaver_preconfigure.
 
-Note
+Note 
 ----
-As per SAP notes 2002167 and 2772999, the role will switch to tuned profile sap-netweaver no matter if another tuned profile
+On RHEL, as per SAP notes 2002167 and 2772999, the role will switch to tuned profile sap-netweaver no matter if another tuned profile
 (e.g. virtual-guest) had been active before or not.
 
+On SLES, this role will switch the saptune solution to the one specified by the configuration and will override any previously set solution.
+The default solution is 
```

Comment:
It appears that some text is missing after "is", to complete this sentence.

Reply:
This was updated to...
> The default solution is `NETWEAVER`